libs/filesystem/pom.xml

```diff
@@ -15,10 +15,61 @@
   </parent>
 
   <properties>
-    <maven.compiler.target>14</maven.compiler.target>
     <maven.compiler.source>14</maven.compiler.source>
+    <maven.compiler.target>14</maven.compiler.target>
+    <dependency.mockito.version>3.7.7</dependency.mockito.version>
   </properties>
```

Comment:
Ensure that the pom.xml in libs has a module entry for filesystem.

Reply:
Yes, it already did!